src/api/spec/features/webui/requests_spec.rb

```diff
@@ -178,25 +187,39 @@
       end
     end
     describe 'for project' do
-      skip
-    end
-    describe 'for invalid project' do
-      skip
+      it 'opens a review' do
+        login submitter
+        visit request_show_path(bs_request.id)
+        click_link 'Add a review'
+        find(:id, 'review_type').select('Project')
+        fill_in 'review_project', with: submitter.home_project
+        click_button 'Ok'
+        expect(page).to have_text("Review for #{submitter.home_project}")
+      end
     end
     describe 'for package' do
-      skip
-    end
-  end
-
-  context 'comments' do
-    describe 'can be created' do
-      skip
-    end
-    describe 'reply' do
-      skip
+      let(:package) { create(:package, project: submitter.home_project) }
```

Comment:
If there is only one example in this describe I would use a local variable instead of a `let`:
```ruby
   package = create(:package, project: submitter.home_project)
```